_inc/client/security/sso.jsx

```diff
@@ -51,46 +53,52 @@ export const SSO = moduleSettingsForm(
 					{ ...this.props }
 					hideButton
 					module="sso"
-					header={ __( 'WordPress.com log in', { context: 'Settings header' } ) }>
+					header={ __( 'WordPress.com log in', { context: 'Settings header' } ) }
+				>
 					<SettingsGroup hasChild disableInDevMode module={ this.props.getModule( 'sso' ) }>
 						<ModuleToggle
 							slug="sso"
 							disabled={ unavailableInDevMode }
-							activated={ isSSOActive }F
+							activated={ isSSOActive }
+							F
 							toggling={ this.props.isSavingAnyOption( 'sso' ) }
 							toggleModule={ this.props.toggleModuleNow }
 						>
-						<span className="jp-form-toggle-explanation">
-							{
-								this.props.getModule( 'sso' ).description
-							}
-						</span>
+							<span className="jp-form-toggle-explanation">
+								{ this.props.getModule( 'sso' ).description }
+							</span>
 						</ModuleToggle>
 						<FormFieldset>
 							<CompactFormToggle
 								checked={ this.state.jetpack_sso_match_by_email }
-								disabled={ ! isSSOActive || unavailableInDevMode || this.props.isSavingAnyOption( [ 'sso', 'jetpack_sso_match_by_email' ] ) }
-								onChange={ () => this.updateOptions( 'jetpack_sso_match_by_email' ) }>
+								disabled={
+									! isSSOActive ||
+										unavailableInDevMode ||
```

Comment:
I think this is consistent because the first line defines a group of statements that constitute a larger statement that returns a boolean. Only in this case it doesn't start with an `if` that's usually located on the previous indentation level, but with the first of several conditions. I'd say it's fine like it is.